package.json

```diff
@@ -16,8 +16,6 @@
   },
   "devDependencies": {
     "@vuepress/bundler-vite": "2.0.0-rc.19",
-    "@vuepress/plugin-markdown-image": "2.0.0-rc.66",
```

Comment:
Are these suposed to be removed or was there a change we expected to see in this file?

Reply:
As far as I can tell, the theme has bundled it internally and no longer needs you to install these plugins yourself.  You can see that the functionality still works due to graphics still being displayed properly: 
![image](https://github.com/user-attachments/assets/d3a4011e-5c8e-4e69-b661-4e160751c198)

![image](https://github.com/user-attachments/assets/4fe3e296-bf5c-4b62-9dfe-a6464064547b)
Here is the package-lock file which shows that the plugin is still installed even after I removed it from our package.json file.